### ISSUE TITLE
Add more options to the dump command

### ISSUE THREAD

**Issue by bensampaio** [willdurand/BazingaJsTranslationBundle#171]
I would like to have some options to control the output of the dump command. Since I couldn't find any on the documentations I decided to leave here some ideas.

Currently when I run `php app/console bazinga:js-translation:dump` I get a `translations` folder with a folder for every domain and inside these folder there is a `js` and a `json` file for every language. I have no use for the `js` files in my project so I would prefer if I could do something like `php app/console bazinga:js-translation:dump --json` in order to get only `json` files.

Would also be helpful if there was a merge option because in my project I want to load all domains at once. Could be something like `php app/console bazinga:js-translation:dump --json --merge`.

What do you think?

**Comment by monteiro** [comment]
@bensampaio hi!
Thank you very much for the issue!

About the first question, that is a good idea. Does the command takes a lot of time to you?
About the second, merging all the domains, that is cool.

If you want, you can create a PR ;)

**Comment by bensampaio** [comment]
@monteiro As you suggested, I created the PR #172. With a few differences from what I initially proposed though. I realised that having a `formats` option would be more adaptable to future versions (in case more formats will be supported) then a `js` and a `json` option. I also thought it would be useful to have a `locales` option to control the languages that are included in the output. With these options, the generation of the files that I actually use in my project became way faster and I see only what I need in the translations folder. Let me know what you think!
